scripts/variantstore/wdl/GvsAssignIds.wdl

```diff
@@ -0,0 +1,168 @@
+version 1.0
+
+workflow GvsAssignIds {
+
+  input {
+    Array[String] external_sample_names
```

Comment:
Would this be more usable as a File that we then do a read_lines on?  Or have both options?